### DISCUSSION TITLE
How to determine issues with InterpreterAndAOT Wasm release

### DISCUSSION THREAD

**Discussion by assassin316** [unoplatform/uno#14629] [General]
Releasing my app in Interpreter mode works fine, but doing so in InterpreterAndAOT never laods. The project successfully compiles but loads in the following state:

![image](https://github.com/unoplatform/uno/assets/17514380/79fc826e-2a1e-41c4-b0f2-639ac4a29468)

How do I go ahead and debug or investigate the source of this problem?

**Comment by jeromelaban** [comment]
Thanks for the question. It looks like you're trying to load the app from a local path, this is not supported by WebAssembly. Try using [`dotnet serve`](https://platform.uno/docs/articles/uno-development/docfx.html?tabs=tabid-1#use-a-local-server) instead.

**Comment by assassin316** [comment]
Worked, perfect!, thanks! 👍
